jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/ExecutionFailedException.java

```diff
@@ -2,11 +2,13 @@
 
 public class ExecutionFailedException extends Exception {
 
+    private static final long serialVersionUID = 1;
```

Comment:
comment about what this variable does

Reply:
SerialVeriondUID is acceptable use in an exception